### PR TITLE
{bio} [dummy] MuTect-1.1.7-Java-1.7.0_80

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)
Update MuTect to 1.1.7

### CODE REVIEW

File: easybuild/easyconfigs/m/MuTect/MuTect-1.1.7-Java-1.7.0_80.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ data of cancer genomes. """
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['http://www.broadinstitute.org/cancer/cga/sites/default/files/data/tools/mutect/']
+sources = ['muTect-%(version)s-bin.zip']
+
+java = 'Java'
+javaver = '1.7.0_80'
+versionsuffix = '-%s-%s' % (java, javaver)
```
Comment (boegel) [inline easybuild/easyconfigs/m/MuTect/MuTect-1.1.7-Java-1.7.0_80.eb]: same here, please use `%(javaver)`, move up `versionsuffix` right below `version`, and clean things up (drop `java` and `javaver` local vars)

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2076.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/727d9e387330deaf003fb4042578d0f9 for a full test report.

**Comment by boegel** [comment]
@verdurin hmm, download failed?

**Comment by verdurin** [comment]
@boegel - sadly, you have to be logged in to download...

**Comment by boegel** [comment]
@verdurin then there's no point in defining `source_urls`?

Just mention the URL as a comment above `sources`?

**Comment by verdurin** [comment]
Closing this PR because of the mess with #2930

**Pull request closed without merging**